### PR TITLE
runtime/parachains: s/twox_64_concat/blake2_128_concat in maps

### PR DESCRIPTION
**Pull request by ordian**
Addresses https://github.com/paritytech/srlabs_findings/issues/74.

I believe we'll be doing proper weights adjustments later.

### CODE REVIEW

**Review by Az111z — APPROVED** [review]

File: primitives/src/v1.rs [hunk 1]
```diff
@@ -112,7 +112,7 @@ pub mod well_known_keys {
 		channel.using_encoded(|channel: &[u8]| {
 			prefix.as_ref()
 				.iter()
-				.chain(twox_64(channel).iter())
+				.chain(blake2_128(channel).iter())
```
Comment (Az111z) [inline primitives/src/v1.rs:115]: وقت التشغيل / parachains: s / twox_64_concat / blake2_128_concat في الخرائط # 2760

**Review by Az111z — APPROVED** [review]

**Review by Az111z — APPROVED** [review]

**Comment by shawntabrizi** [comment]
@gavofyork do we prefer this change? or to have that teams cannot select their ParaId, and instead they are assigned monotonically?

**Review by eskimor — APPROVED** [review]
Approving in the sense, that it seems to do what the issue suggests. Someone could also check for compatibility issues/performance issues - that would be a good idea, I think.

**Comment by gavofyork** [comment]
i don't want every `twox_64_concat` storage item needlessly changed. i'm happy to make `ParaId` be `blake2_128_concat`.

**Comment by ordian** [comment]
@gavofyork I didn't change every `twox_64_concat`. Do you want me to revert the changes for `(T::AccountId, ParaId)` and `HrmpChannelId` keys?

**Comment by ordian** [comment]
(closed in favor of #2988)

**Pull request closed without merging**